crates/server/src/store/in_memory/authority.rs

```diff
@@ -309,6 +309,7 @@ impl InMemoryAuthority {
         }
     }
 
+    #[cfg(feature = "sqlite")]
```

Comment:
This probably will deserve a different flag in the future... but this is ok for now.